caluma/caluma_form/domain_logic.py

```diff
@@ -95,7 +109,7 @@ def create(validated_data: dict, user: Optional[BaseUser] = None) -> models.Answ
 
     @staticmethod
     @transaction.atomic
-    def update(validated_data, answer):
+    def update(answer, validated_data, user: Optional[BaseUser] = None):
```

Comment:
Any reason for the change in parameter ordering? If someone uses this instead of the public API, this will break their code

Reply:
You said it yourself, this is considered a private api, thus it can be subject to change in a non-breaking manner.
The ordering makes things more consistent, which I feel is necessary.